helpers/ts-faker.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "ts-faker",
+  "desc": "Generate fake data using TypeScript interfaces",
+  "url": "https://github.com/SirwanAfifi/ts-faker",
```

Comment:
Heyooo, thanks for contributing. Could we change the URL to this one: https://ts-faker.vercel.app/? :)

Reply:
Thank you @stefanjudis 
just pushed the change.